core/kg/common/utils.go

```
@@ -0,0 +1,41 @@
+package common
+
+import (
+	"bytes"
+	"encoding/base32"
+	"encoding/json"
+	"io"
+	"time"
+
+	"github.com/pborman/uuid"
+)
+
+// MapFromJson decodes the key/value pair map
+func MapFromJson(data io.Reader) map[string]string {
+	decoder := json.NewDecoder(data)
+
+	var res map[string]string
+	if err := decoder.Decode(&res); err != nil {
+		return make(map[string]string)
```

Using this in production does not seem safe.
Why swallow/hide the error?